src/Resources/config/services.xml

```diff
@@ -9,6 +9,9 @@
 
         <service id="webpack_encore.entrypoint_lookup" class="Symfony\WebpackEncoreBundle\Asset\EntrypointLookup">
             <argument /> <!-- entrypoints.json path -->
+            <call method="setCache">
+                <argument key="$cache" type="service" id="webpack_encore.cache" />
```

Comment:
Remove the `$key` here - the whole "binding by named arguments" thing is really something that should only be used in application land. In a bundle, we should avoid this and pass arguments by order - it'll avoid a minor performance impact when rebuilding the container.